components/03-sections/02-navigation/04-side-navigation-news/side-navigation-news.hbs

```diff
@@ -0,0 +1,91 @@
+<!-- Side Navigation -->
+<div class="sidebar-news">
+    {{!-- Related Stories Links --}}
+    <div class="sidebar-news-container">  
+        <div class="sidebar-news-title">Related News</div>
+            <ul class="related-news-items">
+                <li> <span class="related-news-link-icon"></span> <a href="#" aria-label="Related News Link">Link to another related article or promoted news article a really long title for this one.</a></li>
+                <hr>
```

Comment:
I know I told you to add `<hr>`'s, but I'm not sure if it's semantically correct to add anything else besides an `<li>` in an `<ul>`. Maybe we should do margin-bottom instead?